benchmarks/kernels/benchmark_moe.py

```diff
@@ -224,20 +249,15 @@ def sort_config(config: BenchmarkConfig) -> BenchmarkConfig:
     }
 
 
-def save_configs(
-    configs: Dict[int, BenchmarkConfig],
-    num_experts: int,
-    shard_intermediate_size: int,
-    hidden_size: int,
-    topk: int,
-    dtype: torch.dtype,
-    use_fp8: bool,
-) -> None:
-    dtype_str = "float8" if use_fp8 else None
+def save_configs(configs: Dict[int, BenchmarkConfig], num_experts: int,
+                 shard_intermediate_size: int, hidden_size: int, topk: int,
+                 dtype: torch.dtype, use_fp8: bool, use_int8: bool) -> None:
+    dtype_str = "float8" if use_fp8 else "int8" if use_int8 else None
```

Comment:
Let's break this out, stacked ternaries are a bit much. Also consider the rename I posted above
```suggestion
    dtype_str = None
    if use_fp8:
        dtype_str = "float8"
    elif use_int8:
        dtype_str = "int8"
```

Reply:
Thank you, Addressed